### ISSUE TITLE
Bug with small icons?

### ISSUE THREAD

**Issue by TasyaHarrington** [tinalatif/flat.icns#151]
I've been experiencing a bug with the new icons. The full sized icons in my dock look good, but the tiny icons in finder are weird colors and blurry. Not correct at all. I am uploading screenshots of what I mean. Has anyone else encountered this or know how to fix it? It happen if I change the icons both with LiteIcon, or if I go to 'Get Info' and change the icon there. For the folders I am trying to use the generic folder icon. If it matters, I am running OSX 10.10.3

![screen shot 2015-04-28 at 8 14 04 am](https://cloud.githubusercontent.com/assets/7378228/7371580/1d4a71ce-ed7f-11e4-9fe3-6f9d4852ac7d.png)
![screen shot 2015-04-28 at 8 14 32 am](https://cloud.githubusercontent.com/assets/7378228/7371579/1d4a5a40-ed7f-11e4-8d28-5e31cdf2c5a7.png)

**Comment by jameswlane** [comment]
I noticed the same thing but I was using Liteicon

https://www.dropbox.com/s/lsoptqbnnelf1uu/Screenshot%202015-04-28%2009.25.17.png?dl=0

**Comment by TasyaHarrington** [comment]
Yeah, for me it doesn't matter if I use Liteicon or change them manually.

**Comment by tinalatif** [comment]
Just to confirm, you're using the .icns files, not the .pngs?

**Comment by TasyaHarrington** [comment]
Correct.

**Comment by tinalatif** [comment]
Weird. I'll try to reproduce/fix this later today unless someone figures it out first

**Comment by mknrls** [comment]
this issue is only happening with the generic folders or others also?

**Comment by TasyaHarrington** [comment]
Many things. Gimp, Unarchiver, Photos, Download Folder.... It seems to be many of the newer icons

**Comment by wjramos** [comment]
I'll investigate this as well as the mentioned icons were all mine

**Comment by tinalatif** [comment]
Thanks Jordan. Sounds like there's something up with the ai-to-icns script you're using.

**Comment by mknrls** [comment]
been using it also, for the icons i submitted via pull request.

**Comment by wjramos** [comment]
Applied via LiteIcon ![Get Info](https://www.dropbox.com/s/wvseyj8fjs84frf/Screenshot%202015-04-28%2009.01.30.png]

**Comment by wjramos** [comment]
Has anyone else been able to reproduce?

**Comment by wjramos** [comment]
![Unarchiver get info](https://www.dropbox.com/s/baiyiyfif0mks8j/Screenshot%202015-04-28%2013.30.46.png)

**Comment by TheQuantum** [comment]
I’ve also been seeing the same issue in launchpad:
![screen shot 2015-05-20 at 5 46 22 pm](https://cloud.githubusercontent.com/assets/10932585/7730064/71c9a922-ff18-11e4-97fd-029ac94d4e25.png)
Those icons should be (from left to right) Mint, The Unarchiver, Adobe Acrobat

**Comment by wjramos** [comment]
Very strange, I will look into it

**Comment by mcallinder** [comment]
I get the same behavior with .icns files using either LiteIcon or manual replacing. Saw this with some of the apps listed above. Also Clear and Iterm2. Using the .png works fine.

**Comment by wjramos** [comment]
I'm going to try reexporting all of my icons' icns with an updated export script to see if that will resolve the issue

**Comment by ejgisbertm** [comment]
Hello everybody, I can confirm this also happening in my system (10.10.3) either with LiteIcon or dragging and dropping the .icns file in the info window of the app in Finder. I even tried copying and pasting the .icns file itself inside the .app (opening the package, inside the resource folder).

The strange thing here, is that I can only reproduce the error in my Spotlight's results, not anywhere in the Finder, so I'm going to guess that the icon cache is involved. I'll try some system related troubleshooting and report back later.

![spotlight](https://cloud.githubusercontent.com/assets/13053885/8392313/7a3c3bc8-1cb0-11e5-8e2b-a097001b2c1b.png)

Best regards.

**Comment by brianjking** [comment]
I'm also seeing this.

**Comment by brianjking** [comment]
Seems like using the PNG files may have sorted it out for now...

**Comment by wjramos** [comment]
Working with the creator of ICNS export tool here: choffmeister/adobe-illustrator-icnsexport#7
